source/_static/crd.html

```diff
@@ -39,7 +41,8 @@
     class="row-odd attribute-header"
     id="attr-CRD-acsInfoInd">
     <div class="attribute-name">
-      acsInfoInd<a
+      acsInfoInd
+      <a
```

Comment:
I don't think this PR gains anything from this formatting. Is much harder to read and the diff is enormous.